config/src/main/java/io/quarkus/ts/configmap/api/server/Hello.java

```diff
@@ -1,8 +1,5 @@
 package io.quarkus.ts.configmap.api.server;
 
-import io.quarkus.runtime.annotations.RegisterForReflection;
-
-@RegisterForReflection
```

Comment:
Could you elaborate (perhaps in the commit message) on which uses of the annotation can we get rid of? I don't see any pattern.
If it's just me not getting something that should be obvious from e.g. looking at the https://github.com/quarkusio/quarkus/pull/20225 PR, then a simple hint in this discussion would do. Thanks. :)

Reply:
Is not obvious. There are some exceptions as you can see [on this commit ](quarkusio/quarkus#20225)  but in all the other cases I have tested it manually and I realize that the register for reflection annotation was redundant. 

Take a look to [this documentation](https://quarkus.io/guides/writing-native-applications-tips#registering-for-reflection):

```
When building a native executable, GraalVM operates with a closed world assumption. It analyzes the call tree and removes all the classes/methods/fields that are not used directly.
```

So seems that Mandrel is clever enough to see that this code is used directly, so is not removed (and then doesn't need the annotation "@RegisterForReflection"). 

In most of the cases, you will not need this annotation, but in some other cases as "Reflexion" you will need to tag this code by the annotation "@RegisterForReflection"

Reply:
Ok, thanks for the clarification.